include/cuda/std/detail/libcxx/include/chrono

```diff
@@ -2553,9 +2553,9 @@ inline constexpr days year_month_day::__to_days() const noexcept
     const unsigned __dy  = static_cast<unsigned>(__d);
 
     const int      __era = (__yr >= 0 ? __yr : __yr - 399) / 400;
-    const unsigned __yoe = static_cast<unsigned>(__yr - __era * 400);                // [0, 399]
-    const unsigned __doy = (153 * (__mth + (__mth > 2 ? -3 : 9)) + 2) / 5 + __dy-1;  // [0, 365]
-    const unsigned __doe = __yoe * 365 + __yoe/4 - __yoe/100 + __doy;                // [0, 146096]
+    const unsigned __yoe = static_cast<unsigned>(__yr - __era * 400);                  // [0, 399]
+    const unsigned __doy = static_cast<unsigned>((153 * (__mth + static_cast<unsigned>(__mth > 2 ? -3 : 9)) + 2) / 5 + __dy-1);  // [0, 365]
```

Comment:
Why is the outer static_cast needed here?

Reply:
Two problems mainly. `__mth + negative signed integer` is a warning. The assignment ends up being a demotion as well after `xy / 5 + __dy-1`